modern-es6-only.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = [
+	'last 2 Chrome versions',
+	'last 2 Firefox versions',
+	'last 2 Opera versions',
+	'last 2 Edge versions',
+	'Safari >= 10',
+	'iOS >= 10',
+	'Android >= 96'
```

Comment:
Last question, [caniuse](https://caniuse.com/es6) states Android support for 97+

Reply:
I'm quite confident that it stated 96+ when I created this PR. As I understand it, nowadays "Android" is defacto Chromium for Android, and caniuse lists only the newest version for both. I'd speculate that it will only list "98" for Android Browser and Chrome for Android when that Chrome version has been released.

Not sure what to do with that, to be honest. Should we say something like "last 2 Android versions" here as well, as we do for Chrome in general?

Reply:
As discussed, this now uses `last 2 Android versions`.